azure-pipelines.yml

```
@@ -803,6 +804,17 @@ jobs:
     continueOnError: true
     displayName: test-plume-lib.sh
 
+- job: jspecify_conformance
+  dependsOn:
+   - canary_jobs
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
```

```suggestion
    vmImage: 'ubuntu-latest'
  container: wmdietl/cf-ubuntu-jdk21:latest
  steps:
```

To be consistent with all the other builds.